tests/test-lib.sh

```diff
@@ -56,7 +56,7 @@ check_erofs_fsck () {
 
 check_fsverity () {
     fsverity --version >/dev/null 2>&1 || return 1
-    tmpfile=$(mktemp /var/tmp/lcfs-fsverity.XXXXXX)
+    tmpfile=$(mktemp --tmpdir lcfs-fsverity.XXXXXX)
```

Comment:
Specifically this, its probably a better idea to override `$TMPDIR` before `test-lib.sh` is loaded to get fsverify support.

Maybe something like:

```bash
if [[ -z "${COMPOSEFS_FSVERITY_USE_TMPDIR:-}" ]]; then
    TMPDIR="/var/tmp"
fi
```